packages/builders/src/components/textInput/UnsafeTextInput.ts

```diff
@@ -0,0 +1,155 @@
+import { ComponentType, type TextInputStyle, type APITextInputComponent } from 'discord-api-types/v9';
+import { Component } from '../../index';
+import type { TextInputComponent } from './TextInput';
+import isEqual from 'fast-deep-equal';
+
+export class UnsafeTextInputComponent extends Component<
+	Partial<APITextInputComponent> & { type: ComponentType.TextInput }
+> {
+	public equals(other: TextInputComponent | APITextInputComponent): boolean {
```

Comment:
```suggestion
	public equals(other: UnsafeTextInputComponent | APITextInputComponent): boolean {
```
Shouldn't this be `UnsafeTextInputComponent` instead of `TextInputComponent`?